theano/gpuarray/c_code/dnn_conv_base.c

```diff
@@ -208,6 +208,9 @@ static std::string shape(cudnnConvolutionDescriptor_t convDesc)
     int                                 padA[5];
     int                                 strideA[5];
     int                                 dilationA[5];
+    /* Data type configuration. Format: " -<dtype><precision>" with dtype and precision in {h, f, d},
```

Comment:
just to make this more clear use that in the comment:

" -<input dtype><computation dtype>"